docs/index.md

```diff
@@ -16,11 +16,13 @@ environment, *Percona XtraBackup* is designed to make backups a seamless
 procedure without disrupting the performance of the server in a production
 environment. [Commercial support contracts are available](https://www.percona.com/mysql-support/).
 
-!!! important
+**_Important_** 
```

Comment:
Change it to !!! important for us to have the same style for all the admonitions.